optimum/exporters/onnx/model_configs.py

```diff
@@ -218,15 +258,29 @@ class LlamaOnnxConfig(TextDecoderOnnxConfig):
     DEFAULT_ONNX_OPSET = 13
     NORMALIZED_CONFIG_CLASS = NormalizedTextConfig
 
+    @property
+    def inputs(self) -> Dict[str, Dict[int, str]]:
+        common_inputs = super().inputs
+
+        # Refer to GPT2OnnxConfig inputs comment.
+        if not self.no_position_ids and self.task == "text-generation":
+            if self.use_past_in_inputs:
+                common_inputs["position_ids"] = {0: "batch_size"}
+            else:
+                common_inputs["position_ids"] = {0: "batch_size", 1: "sequence_length"}
+        return common_inputs
```

Comment:
Should we make a `TextDecoderWithPositionIdsOnnxConfig` since it seems to be repeated accross many models?